pkg/v2/tkr/util/topology/cluster.go

```diff
@@ -19,15 +20,53 @@ func SetVariable(cluster *clusterv1.Cluster, name string, value interface{}) err
 		return err
 	}
 
-	cVar := clusterVariableForName(cluster, name)
+	cVar := ensureClusterVariableForName(cluster, name)
 	cVar.Value = *jsonValue
 	return nil
 }
 
 // GetVariable gets the value of the cluster variable.
+// Pre-reqs: cluster.Spec.Topology != nil
```

Comment:
Curious: Is `Pre-reqs` an annotation that validates the boolean that follows it? Or is this just a comment?

Reply:
Just a comment. Its purpose is communicating that we're assuming these conditions are true, not checking them.